Library/Homebrew/utils/github.rb

```diff
@@ -228,13 +228,23 @@ def raise_api_error(output, errors, http_code, headers, scopes)
   end
 
   def issues_matching(query, qualifiers = {})
-    uri = ISSUES_URI.dup
+    uri = URI.parse("#{API_URL}/search/issues")
     uri.query = build_query_string(query, qualifiers)
     open(uri) { |json| json["items"] }
   end
 
   def repository(user, repo)
-    open(URI.parse("https://api.github.com/repos/#{user}/#{repo}")) { |j| j }
+    open(URI.parse("#{API_URL}/repos/#{user}/#{repo}")) { |j| j }
+  end
+
+  def search_code(filename: nil, filenames: nil, user: nil, users: nil, &block)
```

Comment:
@reitermarkus Can you revert this for now so it can be shipped as it was and refactor into this form if/when you/we need to use it elsewhere and with tests? Thanks.